### PR TITLE
test 8

### PR DESCRIPTION
**Pull request by snoack**
This change is [Reviewable](https://reviewable-dev-default-rtdb.firebaseapp.com/reviews/snoack/reviewable-test/11)

### CODE REVIEW

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 1 unresolved discussion

___
*[`bitmask-openvpn.py` line 169 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-NfXCFBYAZCITkFD7rq-:-NfXCFBYAZCITkFD7rq0:b9qtc54) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L169)):*
> ```Python
>     return True
> 
> def restart_openvpn(unused):
> ```

some comment

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 1 unresolved discussion

___
*[`bitmask-openvpn.py` line 169 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-NfXCFBYAZCITkFD7rq-:-NfXFElXBLpt5Dlc9s4X:bxk3wd9) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L169)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

some comment
</blockquote></details>

another comment

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, all discussions resolved

___
*[`bitmask-openvpn.py` line 177 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-NflikSn26X-j3USeP-3:-NflikSn26X-j3USeP-4:bb7p34n) ([raw file](https://github.com/snoack/reviewable-test/blob/ad30c52de5f4eae836cbd90b3f55d80f1eb305f4/bitmask-openvpn.py#L177)):*
> ```Python
>         logging.info("PID file does not exist, OpenVPN doesn't seem to be running")
>     except ProcessLookupError:
>         logging.warning("No such process, cannot restart OpenVPN")
> ```

test

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, all discussions resolved

___
*[`bitmask-openvpn.py` line 177 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-NflikSn26X-j3USeP-3:-Nflix-89Js_ZhzEpnLq:b2ni2sh) ([raw file](https://github.com/snoack/reviewable-test/blob/ad30c52de5f4eae836cbd90b3f55d80f1eb305f4/bitmask-openvpn.py#L177)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

test
</blockquote></details>

foo

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 1 unresolved discussion

___
*[`bitmask-openvpn.py` line 178 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-NflkV9p6Dn08ql3f5ha:-NflkV9p6Dn08ql3f5hb:b2ni41j) ([raw file](https://github.com/snoack/reviewable-test/blob/ad30c52de5f4eae836cbd90b3f55d80f1eb305f4/bitmask-openvpn.py#L178)):*
> ```Python
>     except ProcessLookupError:
>         logging.warning("No such process, cannot restart OpenVPN")
> ```

hey

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 1 unresolved discussion

___
*[`bitmask-openvpn.py` line 178 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-NflkV9p6Dn08ql3f5ha:-Nfll-RH-X_ydDvWSJmR:b2yk4yq) ([raw file](https://github.com/snoack/reviewable-test/blob/ad30c52de5f4eae836cbd90b3f55d80f1eb305f4/bitmask-openvpn.py#L178)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

foo
</blockquote></details>

change dispo

File: bitmask-openvpn.py [hunk 1]
```diff
@@ -166,15 +166,15 @@ def update_cert(force=False):
...
         with open(PID_FILENAME, "r") as file:
             os.kill(int(file.read()), signal.SIGHUP)
     except FileNotFoundError:
         logging.info("PID file does not exist, OpenVPN doesn't seem to be running")
     except ProcessLookupError:
-        logging.warning("No such process, cannot restart OpenVPN")
+        logging.warning('No such process, cannot restart OpenVPN')
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
```
Comment (snoack) [inline bitmask-openvpn.py:180]: gh test c

File: README-renamed-in-base.txt [hunk 2]
```diff
@@ -8,4 +8,4 @@ Cras condimentum porttitor ligula, sit amet efficitur lectus finibus ut. Nulla n
 
 In a efficitur dolor. Suspendisse bibendum dui urna, ac consectetur enim imperdiet sit amet. In pharetra eu justo a venenatis. Nullam hendrerit imperdiet augue, a vehicula nibh bibendum a. Nam at est at risus sagittis pellentesque. Duis eu mauris dapibus, euismod ex id, euismod magna. Maecenas diam libero, vehicula eget convallis ut, aliquet id tortor. Nunc ultrices quam tellus, nec consequat mi sollicitudin quis. Curabitur ut turpis eget ligula vehicula sagittis quis sit amet nisi. Vestibulum vel posuere quam, vel cursus nulla. Suspendisse rhoncus lobortis ornare. Vivamus sodales accumsan massa nec varius. Ut lorem libero, bibendum eu sodales eu, fermentum id sem.
 
-renamed in base
```
Comment (snoack) [inline README-renamed-in-base.txt:11]: foo

**Review by snoack — COMMENTED** [review]
> Reviewed 1 of 2 files at r1.
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 1 of 2 files reviewed, 3 unresolved discussions

File: bitmask-openvpn.py [hunk 3]
```diff
@@ -166,15 +166,15 @@ def update_cert(force=False):
     with open(CERT_FILENAME, "wb") as file: file.write(pem)
     return True
 
-def restart_openvpn():
+def restart_openvpn(unused):
     logging.info("Restarting OpenVPN")
```
Comment (snoack) [inline bitmask-openvpn.py:170]: Test comment.

File: bitmask-openvpn.py [hunk 4]
```diff
@@ -166,15 +166,15 @@ def update_cert(force=False):
     with open(CERT_FILENAME, "wb") as file: file.write(pem)
     return True
```
Comment (test-user-96) [inline bitmask-openvpn.py:168]: hey

File: bitmask-openvpn.py (hunk 3 above)
Comment (snoack) [reply bitmask-openvpn.py:170]: foo

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 6 unresolved discussions (waiting on @test-user-96)

___
*[`README-renamed-in-base.txt` line 3 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OAPmxHP9KFHgsJslLmK:-OAPmxHP9KFHgsJslLmL:bb7p34n) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/README-renamed-in-base.txt#L3)):*
> ```help
> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum molestie est eget felis aliquam, vitae tempor sem venenatis. Vestibulum nec finibus velit, ut condimentum ligula. Quisque vitae diam luctus, imperdiet enim vel, mattis purus. Quisque ac mi leo. Donec blandit ligula non lorem lobortis mattis. Maecenas enim libero, tempor a imperdiet volutpat, vulputate et purus. Aenean maximus diam neque, dignissim efficitur sem tristique at. Donec at felis dui. Praesent ut quam mauris. Maecenas congue fringilla sem, et pulvinar nunc accumsan nec.
> 
> Etiam nec hendrerit est. Maecenas mattis nunc eu eros aliquam tincidunt. Donec convallis lacinia ipsum vitae posuere. Cras convallis nulla sagittis purus tempus scelerisque. Integer turpis nulla, viverra in faucibus vel, ultricies quis felis. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Phasellus commodo blandit elit, sed consectetur turpis.
> ```

test

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 5 unresolved discussions (waiting on @test-user-96)

___
*[`bitmask-openvpn.py` line 170 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#gh-1801646173:-OATQblN9UV9MpBGaJei:b2nhzhq) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L170)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

ddd
</blockquote></details>

bar

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 5 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 5 unresolved discussions (waiting on @test-user-96)

File: bitmask-openvpn.py (hunk 3 above)
Comment (snoack) [reply bitmask-openvpn.py:170]: ___
*[`bitmask-openvpn.py` line 170 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#gh-1801646173:-OAu1MUM-j7F3wEIA7wY:b-j5phsv) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L170)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

bar
</blockquote></details>

test reply

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 5 unresolved discussions (waiting on @test-user-96)

File: README-renamed-in-base.txt (hunk 2 above)
Comment (snoack) [reply README-renamed-in-base.txt:11]: inline reply

File: bitmask-openvpn.py [hunk 5]
```diff
@@ -166,15 +166,15 @@ def update_cert(force=False):
...
+def restart_openvpn(unused):
     logging.info("Restarting OpenVPN")
     try:
         with open(PID_FILENAME, "r") as file:
             os.kill(int(file.read()), signal.SIGHUP)
     except FileNotFoundError:
         logging.info("PID file does not exist, OpenVPN doesn't seem to be running")
     except ProcessLookupError:
-        logging.warning("No such process, cannot restart OpenVPN")
+        logging.warning('No such process, cannot restart OpenVPN')
```
Comment (snoack) [inline bitmask-openvpn.py:177]: new inline comment

Comment (snoack) [reply bitmask-openvpn.py:177]: reply

Comment (snoack) [reply bitmask-openvpn.py:177]: reply from reviewable

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 6 unresolved discussions (waiting on @test-user-96)

File: bitmask-openvpn.py (hunk 5 above)
Comment (snoack) [reply bitmask-openvpn.py:177]: reply from reviewable adhoc

File: bitmask-openvpn.py (hunk 4 above)
Comment (snoack) [reply bitmask-openvpn.py:168]: second inline reply

File: bitmask-openvpn.py (hunk 3 above)
Comment (snoack) [reply bitmask-openvpn.py:170]: another inline reply

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 6 unresolved discussions (waiting on @test-user-96)

File: bitmask-openvpn.py (hunk 5 above)
Comment (snoack) [reply bitmask-openvpn.py:177]: another

File: README-renamed-in-base.txt [hunk 6]
```diff
@@ -8,4 +8,4 @@ Cras condimentum porttitor ligula, sit amet efficitur lectus finibus ut. Nulla n
 
 In a efficitur dolor. Suspendisse bibendum dui urna, ac consectetur enim imperdiet sit amet. In pharetra eu justo a venenatis. Nullam hendrerit imperdiet augue, a vehicula nibh bibendum a. Nam at est at risus sagittis pellentesque. Duis eu mauris dapibus, euismod ex id, euismod magna. Maecenas diam libero, vehicula eget convallis ut, aliquet id tortor. Nunc ultrices quam tellus, nec consequat mi sollicitudin quis. Curabitur ut turpis eget ligula vehicula sagittis quis sit amet nisi. Vestibulum vel posuere quam, vel cursus nulla. Suspendisse rhoncus lobortis ornare. Vivamus sodales accumsan massa nec varius. Ut lorem libero, bibendum eu sodales eu, fermentum id sem.
```
Comment (snoack) [inline README-renamed-in-base.txt:9]: testestest

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 10 unresolved discussions (waiting on @test-user-96)

___
*[`bitmask-openvpn.py` line 166 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7Nbho2pIo8JJJI691:-OB7Nbho2pIo8JJJI692:bvsquyz) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L166)):*
> ```Python
> 
>     logging.info("Writing new certificate to %s", CERT_FILENAME)
>     with open(CERT_FILENAME, "wb") as file: file.write(pem)
> ```

comment 1
___
*[`bitmask-openvpn.py` line 175 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7Ndwf1-kSkPHT_Hke:-OB7Ndwf1-kSkPHT_Hkf:bvsquz0) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L175)):*
> ```Python
>             os.kill(int(file.read()), signal.SIGHUP)
>     except FileNotFoundError:
>         logging.info("PID file does not exist, OpenVPN doesn't seem to be running")
> ```

comment 2
___
*[`bitmask-openvpn.py` line 187 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7Nf6DAsa-4afUZwOv:-OB7Nf6DAsa-4afUZwOw:bvsquz1) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L187)):*
> ```Python
>     check_ca()
>     if update_openvpn_config(args.force) | update_cert(args.force):
>         restart_openvpn()
> ```

comment 3

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 13 unresolved discussions (waiting on @test-user-96)

*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7NyRq0M0zHuM-hI3N:-OB7NyRr-BnUP7G4S6oh:bjfk7x4) (no related file):*
new top discussion

___
___
*[`bitmask-openvpn.py` line 0 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7O2Kd4OZMnk51n9B1:-OB7O2Kd4OZMnk51n9B2:b-nw3q7x) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L0)):*
new file level discussion
___
*[`bitmask-openvpn.py` line 165 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7O56R2MMdkIStHh57:-OB7O56R2MMdkIStHh58:bu810oa) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L165)):*
> ```Python
>         return False
> 
>     logging.info("Writing new certificate to %s", CERT_FILENAME)
> ```

new inline discussion

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 13 unresolved discussions (waiting on @test-user-96)

*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7NyRq0M0zHuM-hI3N:-OB7i5XM99TT9CJtFeSD:b2ni2sh) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

new top discussion
</blockquote></details>

foo

___
___
*[`bitmask-openvpn.py` line  at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7O2Kd4OZMnk51n9B1:-OB7i7cf7GXktyhKJfV6:b2nhzhq) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

new file level discussion
</blockquote></details>

bar

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 16 unresolved discussions (waiting on @test-user-96)

*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kZWJ9oMmSsHXBQIb:-OB7kZWJ9oMmSsHXBQIc:b-503h74) (no related file):*
new top

___
___
*[`bitmask-openvpn.py` line 0 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kcyc9tC9l9L3CQF4:-OB7kcyd1zaN3Am7H-ln:bmb0oqn) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L0)):*
new file 
___
*[`bitmask-openvpn.py` line 164 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7keOP64a2YWsE_YHg:-OB7keOP64a2YWsE_YHh:b-hbn9r6) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L164)):*
> ```Python
>         logging.error("HTTP request downloading the certificate failed")
>         return False
> ```

new inline

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 19 unresolved discussions (waiting on @test-user-96)

*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kwrA0o7KPh6BlYuL:-OB7kwrA0o7KPh6BlYuM:b3kr0) (no related file):*
1

___
___
*[`bitmask-openvpn.py` line 0 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7lr_K5vaKv4BS1PuY:-OB7lr_K5vaKv4BS1PuZ:b3kr1) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L0)):*
2
___
*[`bitmask-openvpn.py` line 185 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7lvLZ0Wt0GyE1R6iB:-OB7lvLZ0Wt0GyE1R6iC:b3kr2) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L185)):*
> ```Python
>     args = parser.parse_args()
>     logging.getLogger().setLevel(args.log_level)
>     check_ca()
> ```

3

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 19 unresolved discussions (waiting on @test-user-96)

*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kwrA0o7KPh6BlYuL:-OB7oU6ECgsMY1rY07WA:b2nid68) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

1
</blockquote></details>

top

___
___
*[`bitmask-openvpn.py` line  at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7O2Kd4OZMnk51n9B1:-OB7oWKh3c231h_L5Kqi:b2ni2sh) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

bar
</blockquote></details>

foo

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 19 unresolved discussions (waiting on @test-user-96)

*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kwrA0o7KPh6BlYuL:-OB7ozTF4zL1PwdlxJAG:b3ksf) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

top
</blockquote></details>

d

___
___
*[`bitmask-openvpn.py` line  at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7O2Kd4OZMnk51n9B1:-OB7p-0o4c9fgUMyZNyZ:b3ksu) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

foo
</blockquote></details>

s

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 19 unresolved discussions (waiting on @test-user-96)

foo
___
bar
*[a discussion (2)](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kwrA0o7KPh6BlYuL:-OB7qlQU9f84j5_Xf1m5:b3kr3) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

d
</blockquote></details>

4

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 21 unresolved discussions (waiting on @test-user-96)

foo
___
bar
*[a discussion (2)](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7reFiAVT1GVrv7w3e:-OB7reFiAVT1GVrv7w3f:b2ni2sh) (no related file):*
foo
___
*[`bitmask-openvpn.py` line 0 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7rfyhAFJ5RMJwKfro:-OB7rfyhAFJ5RMJwKfrp:b2nhzhq) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L0)):*
bar

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7tfjC1sqAQIVcopTS:-OB7tfjC1sqAQIVcopTT:b-503h74) (no related file):*
new top
___
*[`bitmask-openvpn.py` line 0 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7the_-m4KBiOxbgNf:-OB7the_-m4KBiOxbgNg:b-d10rgf) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L0)):*
new file

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kwrA0o7KPh6BlYuL:-OBXEFHfB1uSxNzDODZD:b-7k28e3) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

4
</blockquote></details>

reply
___
*[`bitmask-openvpn.py` line  at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7O2Kd4OZMnk51n9B1:-OBXEHDf9M_VmaWpVWrS:bb7o15c) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

s
</blockquote></details>

rply

File: bitmask-openvpn.py (hunk 4 above)
Comment (snoack) [reply bitmask-openvpn.py:168]: rep,ly

**Review by snoack — COMMENTED** [review]
test

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OBXEWwMDupfloRYt12N:bb7p34n)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

File: README-renamed-in-base.txt (hunk 6 above)
Comment (snoack) [reply README-renamed-in-base.txt:9]: test

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

File: README-renamed-in-base.txt (hunk 2 above)
Comment (snoack) [reply README-renamed-in-base.txt:11]: 2

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7NyRq0M0zHuM-hI3N:-OBaXL3B0GcO2_r7NYOh:bt34od8) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

foo
</blockquote></details>

reply 34

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[`bitmask-openvpn.py` line 166 at r1](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7Nbho2pIo8JJJI691:-OBlZq0RClzO7zadBsEE:b2ni2sh) ([raw file](https://github.com/snoack/reviewable-test/blob/92cf055cd9370f3ccdeb4fa025335c2fdc8912e9/bitmask-openvpn.py#L166)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

comment 1
</blockquote></details>

foo

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7reFiAVT1GVrv7w3e:-OC0AeFIALbC_wgCq987:b3ksh) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

foo
</blockquote></details>

f

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7NyRq0M0zHuM-hI3N:-OC0Ai9M0YCGeiOOCL3V:b3ksf) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

reply 34
</blockquote></details>

d

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7NyRq0M0zHuM-hI3N:-OC0CyPz6vS8aKEOVwMB:b3ksf) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

d
</blockquote></details>

d

**Review by snoack — COMMENTED** [review]
d

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OC0EJ-y2djMRmV-QLaz:b3ksf)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
d

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OC0EQ2LEms5lXyB2Z1S:b32wh7)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
d

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OC0F3cb4fTrIyq-aFVw:b3ksf)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
s

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OC0R5CHDzRRv_TIOP-Y:b3ksu)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
s

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OC0RAly-q2qni7O2vPW:b3ksu)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
d

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OC0UYcuAREjpwFpQTKI:b3ksf)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
d

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OC0Uag0AaQi_3RWf7Im:b3ksf)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kZWJ9oMmSsHXBQIb:-OCGQusl6wGL0Kjrr_tk:b3ksg) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

f
</blockquote></details>

e

**Review by snoack — COMMENTED** [review]
3

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OCGRDZA5NWWEk9FnTz9:b3kr2)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

___
*[a discussion](http://localhost:8080/reviews/snoack/reviewable-test/11#-OB7kZWJ9oMmSsHXBQIb:-OCGsokO7h9tU4nlIVCG:b2ni1sw) (no related file):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

e
</blockquote></details>

eee

**Review by test-user-96 — APPROVED** [review]
g

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OCGtoHz15-xiAfpzrb_:b3ksi)* status: 0 of 2 files reviewed, 23 unresolved discussions

**Review by test-user-96 — CHANGES_REQUESTED** [review]
e

**Review by snoack — COMMENTED** [review]
d

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/11#-:-OCGt76V4ucwvTwxaA-h:b3ksf)* status: 0 of 2 files reviewed, 23 unresolved discussions (waiting on @test-user-96)

**Review by snoack — COMMENTED** [review]
foo

**Pull request closed without merging**